### PR TITLE
Email and ZIP Validation

### PR DESCRIPTION
**Pull request by daytonbrock**

### CODE REVIEW

**Review by mkbaker — COMMENTED** [review]
Great work!

File: src/components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -60,6 +61,8 @@ class LandingPage extends Component {
               <VoteButtonModal />
             </div>
             {this.props.reduxState.voteSubmissionModalReducer ? <VoteSubmissionModal></VoteSubmissionModal> : <></>}
```
Comment (mkbaker) [inline src/components/LandingPage/LandingPage.js:64]: I think we should add comments describing any ternary operators -- we can totally add that later though

**Review by mkbaker — DISMISSED** [review]
👍

File: src/components/LandingPage/LandingPage.js (hunk 1 above)
Comment (mkbaker) [reply src/components/LandingPage/LandingPage.js:64]: This is also a note to myself lol

File: src/components/ReturningVoterMessage/ReturningVoterMessage.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  };
+
+  render() {
+
+    return (
+      <div>
+        <Modal size="mini" open={this.state.modalOpen}>
+          <Modal.Content>
+            <h4>
+              You have already voted and may not vote twice.
```
Comment (mkbaker) [inline src/components/ReturningVoterMessage/ReturningVoterMessage.js:18]: Just a thought on wording: "Our records indicate you have already voted." ?

Comment (daytonbrock) [reply src/components/ReturningVoterMessage/ReturningVoterMessage.js:18]: that sounds good. was struggling with the copy, so thanks for catching that.

**Review by mkbaker — APPROVED** [review]
👍 👍

**Pull request merged**